src/library/simplified/extension/marketing-ab-tests.js

```diff
@@ -0,0 +1,16 @@
+utag_data['ABTestsExpValues'] = '';
+var tpid = utag_data['context.site.tpid'];
+var eapid = utag_data['context.site.eapid'];
+var guid = utag_data['guid'];
+var domain_name = utag_data['context.site.siteName'];
```

Comment:
There may be a possibility of CORS(Cross0riginResourceSharing) issue if we try to get response from expedia.com for trunk/int url.
In production we may not experience this issue, but in lab url's we will get this error in the browser console.
Error desc:  No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://wwwexpediacom.trunk.sb.karmalab.net' is therefore not allowed access.

This is because, we are trying to call prod abacus API from trunk url which creates CORS issue.
In order to come out from this issue, we have to use the corresponding url's. 
Please suggest me on this